src/views/projects/Cob/AssigneeInput.svelte

```diff
@@ -22,10 +22,11 @@
 
   function addAssignee() {
     if (parsedNodeId) {
-      if (updatedAssignees.includes(parsedNodeId.pubkey)) {
+      const assignee = `${parsedNodeId.prefix}${parsedNodeId.pubkey}`;
+      if (updatedAssignees.includes(assignee)) {
         validationMessage = "This assignee is already added";
       } else {
-        updatedAssignees = [...updatedAssignees, parsedNodeId.pubkey];
+        updatedAssignees = [...updatedAssignees, assignee];
```

Comment:
This should be did-formatted

Reply:
It is already.

Reply:
It is. We run it through the `parseNodeId` function above to make sure it's a valid node id and then put it back together on line 25.